tests/vmi_cloudinit_test.go

```diff
@@ -195,7 +195,7 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 						libvmi.WithCloudInitConfigDriveUserData(userData),
 					)
 
-					LaunchVMI(vmi)
+					vmi = LaunchVMI(vmi)
```

Comment:
Please squash as the reason for this commit is the first commit...

Reply:
@xpivarc you're the approver, so I accept your verdict.

Just a personal note: I would have accepted the second commit even if it was standing alone, as it makes sense to always use the most recently known VMI. But strictly speaking, it should have had a different subject and to come first.

Reply:
Squashed.

Reply:
and rebased

Reply:
> In addition, retrieve the return value of LaunchVMI(vmi) function for all callers, to avoid using a VMI with an empty namespace and always use the most recently known VMI.

Well, if the second commit was doing this, then it should have been a dedicated commit for sure.
But it does not worth the debate now.